src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java

```diff
@@ -423,11 +425,26 @@ public void visit(Parenthesis parenthesis) {
     @Override
     public void visit(StringValue stringValue) {
         if (stringValue.getPrefix() != null) {
-            buffer.append(stringValue.getPrefix());
+            buffer.append(
```

Comment:
Prefix is related to encoding like utf_8'string' and therefore doesn't need quotation mark support.

Reply:
Okay i will remove it